### ISSUE TITLE
Request to Recompile

### ISSUE THREAD

**Issue by nivthefox** [Draylar/identity#647]
Hello!  I am the mod author for Arcane Additions, a companion mod to the Mana and Artifice mod which adds new spells and compatibilities. One of the compatibilities I support is a "Polymorph" component, which utilizes the Identity mod to allow players to transform into animals using the Mana and Artifice mod.

Recently, Mana and Artifice updated to support 1.20.1, using Forge version 47.1.46.  However, it seems that Identity does not work with that Forge version due to a change in how Mixins work.  

I know this is a lot to ask, and I will respect your answer if you outright say no, but it'd be fantastic to get a new version of Identity compiled for something in the 47.2 range of Forge, or Neoforged, so that I can make this compatibility work again.  Otherwise I'll have to go look for an alternative shapechanging mod to compat with, and I was very happy with how Identity worked for this.

Please let me know!

Thanks!

**Comment by ToCraft** [comment]
You *could* use my fork, it works for 1.20.+ . (though I renamed basically everything, it's still a similar API). It'S way more advanced then identity, too.

But, I you really want to stay with Identity, I think it should be okay to implement it into your mod, though you need some note and reference to the original mod.

https://curseforge.com/minecraft/mc-mods/remorphed
https://curseforge.com/minecraft/mc-mods/woodwalkers

**Comment by nivthefox** [comment]
> You _could_ use my fork, it works for 1.20.+ . (though I renamed basically everything, it's still a similar API). It'S way more advanced then identity, too.

I was looking at yours, but I am honestly sadenned that I have to find a new provider every two years for this. I'm debating just forking it myself.

Question for yours: does it support disabling the means of acquiring morphs at the server level?

**Comment by ToCraft** [comment]
> Question for yours: does it support disabling the means of acquiring morphs at the server level?

Yeah, I basically just extended the API of Identity, so everything that worked for Identity should work for my mod, too, after some renaming (e.g. I renamed the packages from `draylar.identity` to `tocraft.walkers` and renamed `Identity` to `Walkers` or `Shape`). Furthermore, disabling the unlocking mechanics (or implementing new mechanics) definitely works, since I wrote "[Remorphed](https://curseforge.com/minecraft/mc-mods/remorphed)" which effectively does that.
